.github/ISSUE_TEMPLATE/1-bugreport.yml

```diff
@@ -0,0 +1,55 @@
+name: "Bug Report"
+description: "File a bug report"
+title: "[Bug]: "
+labels: ["bug"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thank you for taking the time to fill out this bug report!
+  - type: input
+    id: version
+    attributes:
+      label: "Version"
+      description: "Which node version are you running?"
+      placeholder: "v0.1.0"
+    validations:
+      required: true
+  - type: textarea
+    id: other-version
+    attributes:
+      label: "Other packages versions"
+      description: "Let us know of the versions of any other packages used. For example, which version of the client are you using?"
+      placeholder: "miden-client: v0.1.0"
+    validations:
+      required: false
```

Comment:
Should this be more about "protocol version" rather than about "note version"?

Also, I wonder whether we should just have a single text area for describing all relevant version info.

Reply:
merged